### PR TITLE
Fix watch env vars

### PR DESCRIPTION
**Pull request by scottkurz**
* Fixes formatting of `true|false` descriptions
* Fixes formatting of watch regex
* Also, I think the cause and effect isn't clear w.r.t. the ON_CHANGE vars.   As I understand it, if you blank out those values, then file watching won't happen.   But you could have non-blank values, and run `appsody run --no-watcher`...nothing wrong with that right?

### CODE REVIEW

**Comment by skoh7645** [comment]
LGTM, once @helenmasters takes a look at the wording it can go in

File: content/docs/stacks/environment-variables.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The environment variables that define how to start the development container, th
...
-| `APPSODY_RUN_ON_CHANGE=<command>`   | **Optional**. This command runs when a change is detected on the file system by the controller. <br><br> If file watching is disabled, this value should be "". | ` APPSODY_RUN_ON_CHANGE="npm start"` <br><br> Runs `npm start` when a file being watched by the controller is changed.                                                                                          |
+| `APPSODY_RUN_ON_CHANGE=<command>`   | **Optional**. This command runs when a change is detected on the file system by the controller. <br><br> If this value is set to "", file watching will be disabled. | ` APPSODY_RUN_ON_CHANGE="npm start"` <br><br> Runs `npm start` when a file being watched by the controller is changed.                                                                                          |
```
Comment (helenmasters) [inline content/docs/stacks/environment-variables.md]: Scott, thanks for raising this - although I can't believe you've viewed this file in GitHub!!! (yuck)

One v minor change - please can you change "will be disabled" to "is disabled" as we try and avoid using future tense in the docs.

File: content/docs/stacks/environment-variables.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The environment variables that are used to set up the development workspace for
...
+| `APPSODY_USER_RUN_AS_LOCAL=[true\|false]`         | **Optional**. Sets the user identity within the container to match the host user’s UID. If a stack mounts a host directory, it ensures that files written to the host directory are using the host user’s UID, to avoid container-created files being owned by `root`. <br><br> The default is `false`.                                                                                                                                                                                                                                                                                                                                                                               | ` APPSODY_USER_RUN_AS_LOCAL=true` <br><br> The container's user identity is set to the host user's UID.                                                                                                                                                                                   |
```
Comment (helenmasters) [inline content/docs/stacks/environment-variables.md]: The problem with this change, as far as I can tell, is that it fixes the file displayed in GitHub but it breaks the website view as this page on appsody.dev then displays [true\|false]

Comment (scottkurz) [reply content/docs/stacks/environment-variables.md]: Oh yeah.. I forgot what this was the source for :)  Blame Google for sending me here.    I guess I can still submit the rewording part.

Comment (helenmasters) [reply content/docs/stacks/environment-variables.md]: Yes that'd be great, thanks.

File: content/docs/stacks/environment-variables.md (hunk 1 above)
Comment (scottkurz) [reply content/docs/stacks/environment-variables.md]: Thanks for acknowledging my pain in reading markdown ;)     Alright I re-pushed with just the rephrasing, not the markdown change (with your suggestion)

**Pull request merged**